crates/autopilot/src/domain/settlement/solution/mod.rs

```diff
@@ -194,6 +200,16 @@ pub mod error {
     }
 }
 
+/// Fee per trade in a solution. Contains breakdown of protocol fees and total
+/// fee.
+#[derive(Debug, Clone)]
+pub struct OrderFee {
```

Comment:
nit:

```suggestion
pub struct ExecutedFee {
```

maybe even allowing for nice name-spacing `order::ExecutedFee` or `order::executed::Fee`

Reply:
renamed